eservice/pdo/eservice/scripts/EServiceCLI.py

```diff
@@ -247,7 +247,7 @@ def LocalMain(config) :
 ContractKeys = os.path.join(ContractHome, "keys")
 ContractLogs = os.path.join(ContractHome, "logs")
 ContractData = os.path.join(ContractHome, "data")
-LedgerURL = os.environ.get("PDO_LEDGER_URL", "http://127.0.0.1:8008/")
+LedgerURL = os.environ.get("PDO_LEDGER_URL", "http://127.0.0.1:6600/")
```

Comment:
Thing is that, with CCF and the certificates assigned to specific IPs and/or hostnames, you cannot really have default values.
In fact, it makes sense that the ledger endpoints are something that you figure out at runtime and share the information with interested parties.

Reply:
> Not sure I understand, we set the ledger URL to `'http://127.0.0.1:6600'` everywhere if `PDO_LEDGER_URL` is not set... that seems to be the default. 

Correct.
I am saying that with CCF these default values are unusable -- because the subject name in CCF certificates will likely not match such default value.

> All I was suggesting was that instead of copy-pasting this URL in every python script, we should define a constant `DEFAULT_LEDGER_URL = 'http://127.0.0.1:6600'` in one place, so that when that changes, we only need to edit a single place, not a bunch of individual scripts.

I agree that this consolidation is practical, and actually usable (at least) with Sawtooth.